src/xdocs/config_misc.xml

```diff
@@ -2197,7 +2197,22 @@ messages_home_fr_CA_UNIX.properties
         <source>
 &lt;module name=&quot;UncommentedMain&quot;/&gt;
         </source>
+        <p>
+          Example:
+        </p>
```

Comment:
please change it to: `<p>Example:</p>`

Reply:
@AmrDeveloper , please do update as requested by @baratali .